src/request/request-options.js

```diff
@@ -45,6 +46,10 @@ const getRequestOptions = (endpointOptions = {}) => {
     url = addQueryParameters(url, paramGroups.query)
   }
 
+  if (produces.length > 0) {
+    headers['accept'] = produces
```

Comment:
Are you sure headers['accept'] supports array?

Reply:
@MunifTanjim not sure wbout the debug output, but http accept headers definitely support multiple values just like content-type.

We should be doing whatever that does. I'm guessing either an array or possibly tokenizing a string